### PR TITLE
XRWH-37 Updated style for headers

### PR DESCRIPTION
**Pull request by vasyl-krupa**

### CODE REVIEW

**Review by yuliia-biletska — APPROVED** [review]
approved

File: plugins/rocks.xml.webhelp/css/topic.css [hunk 1]
```diff
@@ -35,6 +35,13 @@
 :root {
     --bs-font-sans-serif: 'Montserrat', sans-serif;
 }
+
+h1.title {
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/css/topic.css:19]: What about 2-nd+ level topic title? Please update input content for testing by adding chunk attr.

Comment (vasyl-krupa) [reply plugins/rocks.xml.webhelp/css/topic.css:39]: @ibarinova  We need to discuss this together

**Comment by vasyl-krupa** [comment]
http://uarnet.intelliarts.com/vkrupa/Chunk604918239.html#Chunk604918239 -example

Please note that I also pushed fixes for monospace font, incorrect path in chunked topics and missing styles from pprevious merges.

**Review by ibarinova — APPROVED** [review]
Very good.

**Pull request merged**